src/mock/dialog.ts

```diff
@@ -4,6 +4,7 @@ import { JSONArray, JSONObject, ResponseData } from 'yet-another-fetch-mock';
 import bruker from './bruker';
 
 const dialoger: DialogData[] & JSONArray = [
+    //   ]; const dialogs : DialogData[] & JSONArray = [ // kommenter når "ingen dialoger" er testet ferdig
```

Comment:
Tenker den kommentaren burde enten skrives om, eller fjernes. for akkurat nå så gir den ingen verdi.

Bedre hvis det står noe ala: 
` // eks på ingen dialoger`

Reply:
gjort